packages/webapp/components/CookieBanner.tsx

```diff
@@ -40,7 +40,7 @@ export default function CookieBanner({
       <ModalCloseButton onClick={close} />
       <CookieIcon
         className="hidden laptop:block mb-4 text-theme-label-primary"
-        style={{ fontSize: '3.5rem' }}
+        style={{ fontSize: '3.5rem', width: 'auto', height: 'auto' }}
```

Comment:
Now we don't have it, and it's a little bit different thing - if we want to go with a custom size through fontSize we need to reset width and height, because they are always set inside icon with some tailwind classes

Reply:
I would suggest adding a size property that adds these auto values so we can be consistent with how we set the width and height.

Reply:
What about xxxlarge size prop for this one? It will be w-14 h-14? So no fontSize at all

Reply:
That can work, if we it is deemed to be the actual next size. If not I would suggest just override the height and width. But long as we are still using the height and width, I think we are consistent enough.

Reply:
Really this size will be too specific and used only in one place, so let's go with auto for now

Reply:
I would prefer fixed sizes as well, if we don't have them we should add them.
I think the xxxlarge can work, I added that recently for a similar approach.

Reply:
Done